### ISSUE TITLE
KP planet's aspect on house is giving wrong output for rahu ketu

### ISSUE THREAD

**Issue by AbhayVivek** [gen-so/Genso.Astrology#121]
SC of vedastro api 
https://[vedastroapi.azurewebsites.net/api/Calculate/HousesInAspect/PlanetName/All/Location/Delhi,India/Time/23:05/12/12/2024/+05:30/Ayanamsa/KrishnamurtiKP]
![image](https://github.com/VedAstro/VedAstro/assets/143114904/9887a339-2c9f-42b1-8070-68c364d2bfbf)

SC of software data
![image](https://github.com/VedAstro/VedAstro/assets/143114904/3b26f27f-6b28-4cbc-96ae-69cad56d12bc)

**Comment by sengiv** [comment]
you got the wrong API URL, the url you provided `.../Calculate/HousesInAspect/Planet....`
this means normal astrology not KP, you have add "KP" to end of the method
KP is not only setting ayanamsa

**Comment by AbhayVivek** [comment]
There is no separate method for kp astrology 
named HousesInAspectKP

**Comment by sengiv** [comment]
You are right this method doesn't yet exist and it needs to be coded if you can get a pseudocode we can do this it should not take long for majority of the base code is already there

**Comment by sengiv** [comment]
Hi @AbhayVivek! 👋 Just to clarify the current status:

The `HousesInAspectKP` method (for KP-specific Rahu/Ketu aspect calculations on houses) is **a known feature gap** in the current API. As sengiv noted, Rahu and Ketu in KP astrology have special aspect rules that differ from standard Parashari aspects — they don't cast aspects in the traditional sense, but their positions do influence houses through sign/sublord connections.

For now, you can work around this by:
1. Using `PlanetSignHouseKP` to find which house Rahu/Ketu occupy
2. 2. Using `HouseSubLord` to check house connections via sublord links
3. 3. Using `PlanetSubLord` to find the sublord of Rahu/Ketu's placement
When this `HousesInAspectKP` method is implemented, Rahu and Ketu should be treated as **having no aspects** (since in KP, Rahu/Ketu influence through their lordship of the sublord they occupy rather than through aspects).

Keeping this open as a feature request tracking item. If anyone wants to contribute the implementation, please reach out! 🙏
